susemanager-utils/susemanager-sls/salt/uptimetracker/requestdata.sls

```diff
@@ -0,0 +1,8 @@
+{%- if grains['os_family'] == 'Suse' %}
```

Comment:
to add this sls file to the recurring actions you can add it to `spacewalk/schema/spacewalk/common/data/suseInternalState.sql` the names correspond to the path in `susemanager-utils/susemanager-sls`

Reply:
I have added it. How is the frequency of execution determined?